File: Client/Assets/Scripts/Common/AuxComponent/AuxUIComponent/AuxInputField.cs
using UnityEngine;
using UnityEngine.UI;

namespace SDK.Common
{
    public class AuxInputField : AuxComponent
    {
        protected InputField m_inputField;     // 输入

        public AuxInputField(GameObject pntNode, string path, BtnStyleID styleId = BtnStyleID.eBSID_None)
        {
            m_selfGo = UtilApi.TransFindChildByPObjAndPath(pntNode, path);
            m_inputField = UtilApi.getComByP<InputField>(pntNode, path);
        }

        public string text
        {
            get
            {
                return m_inputField.text;
            }
            set
            {
                m_inputField.text = value;
            }
        }
    }
}

File: Client_Start/Assets/uLua/Lua/MyLua/Libs/AuxComponent/AuxUIComponent/AuxInputField.lua
require "MyLua.Libs.Core.GlobalNS"
require "MyLua.Libs.Core.Class"
require "MyLua.Libs.Core.GObject"
require "MyLua.Libs.AuxComponent.AuxUIComponent.AuxWindow"

local M = GlobalNS.Class(GlobalNS.AuxWindow);
M.clsName = "AuxInputField";
GlobalNS[M.clsName] = M;

function M:ctor(...)
    local pntNode, path, styleId = ...;
    if(styleId == nil) then
        styleId = BtnStyleID.eBSID_None;
    end
    self.m_selfGo = GlobalNS.UtilApi.TransFindChildByPObjAndPath(pntNode, path);
    self.m_inputField = GlobalNS.UtilApi.getComByPath(pntNode, path, 'InputField');
end

function M:setText(value)
    self.m_inputField.text = value;
end

function M:getText()
    return self.m_inputField.text;
end

return M;